cli/cmd/plugin/ui/web/tanzu-ui/node-server/main.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+/* globals __dirname */
+const _ = require('lodash');
+const mkdirp = require('mkdirp');
+const paths = require('./src/conf/paths');
+const appConfig = require(paths.src.appConfig);
+const winston = require('winston');
```

Comment:
should we follow the same rule to arrange the imports?
e.g.: 
// third party imports
// app imports

if so, please modify all files below

Reply:
good suggestion